.travis.yml

```diff
@@ -12,10 +12,11 @@ before_script:
   - composer self-update
   - composer update --prefer-source
 
-script: phpunit
+script: vendor/bin/phpunit
```

Comment:
Travis comes with a pre-installed version of phpunit. We want to ensure that all our tests uses the version that is on the composer.json